test/logflare_web/controllers/source_controller_test.exs

```diff
@@ -417,34 +408,16 @@ defmodule LogflareWeb.SourceControllerTest do
     end
   end
 
-  describe "show" do
-    setup [:old_setup, :expect_user_plan]
-
-    test "returns 403 for a source not owned by the user", %{
-      conn: conn,
-      users: [_u1, u2 | _],
-      sources: [s1 | _]
-    } do
-      conn =
-        conn
-        |> login_user(u2)
-        |> get(source_path(conn, :show, s1.id))
-
-      assert html_response(conn, 403) =~ "403"
-      assert html_response(conn, 403) =~ "Forbidden"
-    end
-  end
-
   describe "create" do
-    setup [:old_setup, :expect_user_plan]
+    setup [:create_plan, :old_setup, :expect_user_plan]
```

Comment:
would be good if we can refactor away the :old_setup which makes each test case less straightforward and harder to read